src/test/java/core/basesyntax/io/FileReaderImplTest.java

```diff
@@ -0,0 +1,51 @@
+package core.basesyntax.io;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.Arrays;
+import java.util.List;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Test;
+
+class FileReaderImplTest {
+    private FileReader fileReader;
+    private Path tempFile;
+
+    @BeforeEach
+    void setUp() {
+        fileReader = new FileReaderImpl();
+        tempFile = Path.of("src/test/java/core/basesyntax/resources/test.csv");
```

Comment:
invalid resources folder path. Resources folder should stay on the same level as 'java' folder. The proper path:
```suggestion
        tempFile = Path.of("src/test/resources/test.csv");
```